src/Chapter1.hs

```diff
@@ -209,31 +210,29 @@ So, the output in this example means that 'False' has type 'Bool'.
 > Try to guess first and then compare your expectations with GHCi output
 
 >>> :t True
-<INSERT THE RESULT INSTEAD OF THE TEXT>
+True :: Bool
 >>> :t 'a'
-<INSERT THE RESULT INSTEAD OF THE TEXT>
```

Comment:
I see that doctests are failing on this line because of extra space. Other lines also have some extra spaces. Maybe you can try to remove them, run tests locally and see, where exactly tests are failing?